src/protocol/thread/thread.ts

```diff
@@ -925,7 +927,7 @@ class _ThreadFront {
   }
 
   async findConsoleMessages(
-    onConsoleMessage: (pause: Pause, message: Message) => void,
+    onConsoleMessage: (pause: Pause, message: WiredMessage) => void,
```

Comment:
Why are we still passing `pause: Pause` as the first param?

The only place we use `ThreadFront.findConsoleMessages` – we add a wrapper function that ignores the first param.

Reply:
I don't know, I just didn't want to start refactoring stuff in this PR.